superset/config.py

```diff
@@ -320,6 +320,8 @@ def _try_json_readsha(filepath: str, length: int) -> str | None:
 # ex: http://localhost:8080/swagger/v1
 FAB_API_SWAGGER_UI = True
 
+FAB_ADD_SECURITY_API = True
```

Comment:
@dpgaspar did we make and update in flask app builder to make this default True? When I recently pulled these were getting `KeyError: FAB_ADD_SECURITY_API not found` Also I saw that flask app builder has this default True so to allow these test to run I just made this value consistent with what I saw upstream.

Can you verify that this is what's happening?

Reply:
Default is false on FAB: https://github.com/dpgaspar/Flask-AppBuilder/blob/2d527aae09eed50fd52c6bf886a2970adb225428/flask_appbuilder/security/sqla/manager.py#L97

The link your mentioning is an example app